arm-redis/2017-02-01/swagger/redis.json

```diff
@@ -0,0 +1,1462 @@
+      "get": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "PatchSchedules_Get",
+        "x-ms-examples": {
+          "RedisCachePatchSchedulesGet": { "$ref": "../examples/RedisCachePatchSchedulesGet.json" }
+        },
+        "description": "Gets the patching schedule of a redis cache (requires Premium SKU).",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the redis cache."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of get patch schedules.",
+            "schema": {
+              "$ref": "#/definitions/RedisPatchSchedule"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/linkedServers/{linkedServerName}": {
+      "put": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "RedisLinkedServer_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "RedisLinkedServer_Create": { "$ref": "../examples/RedisCacheLinkedServer_Create.json" }
+        },
+        "description": "Adds a linked server to the Redis cache (requires Premium SKU).",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache."
+          },
+          {
+            "name": "linkedServerName",
+            "in": "path",
+            "type": "string",
+            "description": "The name of the linked server that is being added to the Redis cache."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RedisLinkedServerCreateParameters"
+            },
+            "description": "Parameters supplied to the Create Linked server operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
Please use 201 + provisioningState + optional AzureAsync operation header pattern for PUT/PATCH async operations.